library/alloc/benches/btree/map.rs

```diff
@@ -349,11 +349,13 @@ pub fn iter_100(b: &mut Bencher) {
 }
 
 #[bench]
+#[cfg_attr(target_arch = "wasm32", ignore)] // not enough memory
```

Comment:
Hm, this seems quite arbitrary. Doesn't WASM have `memory.grow` which should be invoked by the allocator to allocate memory pages past the initial allocation?

I feel like this ignore is ignoring a potentially genuine problem or that this test fails to work on 32-bit targets in general and should be rewritten to work on these targets (or ignored if pointer size is 32-bit.)

Reply:
By default, Emscripten doesn't use `memory.grow`; it needs to be [enabled via a flag](https://emscripten.org/docs/optimizing/Optimizing-Code.html#memory-growth). Maybe Rust should enable that flag by default?

Reply:
Sounds to me like it might be a good idea, but also sounds like out of scope for this specific PR.